docs/deployment/deploy-without-docker/Import-the-Seed-Database.md

```diff
@@ -39,11 +39,12 @@ After download, the files can be unzipped by entering the following command:
 
 4. (optional : support for microRNA genomic profiles) Import constructed gene table records for microRNA genomic profiles. Currently, cBioPortal supports the combined display of copy number alterations (generally reported for microRNA precursors) and expression (generally reported for microRNA mature forms) by adding gene table records which represent the combination of microRNA precursor and microRNA mature form. Appropriate aliases are added to the gene_alias table so that both the name of the precursor and the name of the mature form are recognized references to the combination.
 
-    After the code has been successfully configured and built, you can import the needed microRNA records by running the following command from the $PORTAL_HOME directory:
+This involves downloading the cBioPortal Core code located [here](https://github.com/cBioPortal/cbioportal-core). 
```

Comment:
```suggestion
The next step requires downloading the cBioPortal Core code located [here](https://github.com/cBioPortal/cbioportal-core). 
```

As an aside, prolly better to use provided jar here? E.g. from jitpack or another place? Not sure if we want to advocate building `cbioportal-core` as well for the non-docker based setup

Reply:
I would say, it is probably best for them to download the jar module. Building from source is not hard and if they are a savvy enough user they can figure it out themselves. :-)